staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -124,20 +118,12 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 	}
 
 	subnetworkURL := g.SubnetworkURL()
-	if g.AlphaFeatureGate.Enabled(AlphaFeatureILBCustomSubnet) {
-		// If this feature is enabled, changes to subnet annotation will be
-		// picked up and reflected in the forwarding rule.
-		// Removing the annotation will set the forwarding rule to use the default subnet.
-		if options.SubnetName != "" {
-			subnetworkURL = gceSubnetworkURL("", g.networkProjectID, g.region, options.SubnetName)
-		}
-	} else {
-		// TODO(84885) remove this once ILBCustomSubnet goes beta.
-		if existingFwdRule != nil && existingFwdRule.Subnetwork != "" {
-			// If the ILB already exists, continue using the subnet that it's already using.
-			// This is to support existing ILBs that were setup using the wrong subnet - https://github.com/kubernetes/kubernetes/pull/57861
-			subnetworkURL = existingFwdRule.Subnetwork
-		}
+	// If this feature is enabled, changes to subnet annotation will be picked up and reflected in the forwarding rule.
```

Comment:
Sorry dumb question - did the annotation graduate to beta/ga or is this removing it completely? Is this comment still valid?

Reply:
we graduated this support to GA with the annotation intact. So now cleaning up the annotation. Good point, I will remove the comment here.

Reply:
Ah got it, thanks for the explanation.

Reply:
fixed the comment @MrHohn